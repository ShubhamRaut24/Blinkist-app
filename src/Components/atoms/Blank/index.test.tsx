
import BlankComponent from '.'
import renderer from 'react-test-renderer';

test('Blank Component', () => {
    const component = renderer.create(
            <BlankComponent/>
    );
    expect(component).toMatchSnapshot();
});