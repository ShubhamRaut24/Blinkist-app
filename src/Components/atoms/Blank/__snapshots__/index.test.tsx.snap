// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blank Component 1`] = `
<div
  className="MuiBox-root css-iu3tcm"
  data-testid="blankId"
>
  A
</div>
`;
